### PR TITLE
Fix a problem

### PR DESCRIPTION
**Pull request by gov2**
Fix a problem that custom table view cell(with several subviews) caused.

### CODE REVIEW

File: TimerScroller/TimeScroller.m [hunk 1]
```diff
@@ -398,7 +398,9 @@ - (void)scrollViewDidScroll
     point = [_scrollBar convertPoint:point toView:_tableView];
     
     UIView *view = [_tableView hitTest:point withEvent:nil];
-
```
Comment (jon513) [inline TimerScroller/TimeScroller.m:401]: I also fixed this bug.  I used
    UITableViewCell\* cell=[_tableView cellForRowAtIndexPath:[_tableView indexPathForRowAtPoint:point]];
Which I think is a lot nicer.  It uses the built in tableView properties for finding a  tableViewCell given a point.

Comment (gov2) [inline TimerScroller/TimeScroller.m:401]: thx,I'm a newbie, that's helpful

Comment (andrewroycarter) [inline TimerScroller/TimeScroller.m:401]: Jon513, If you could submit a pull request with that change I'd be happy to merge it in!

**Comment by andrewroycarter** [comment]
Merged in Jon513's fix. Thanks!

**Pull request closed without merging**